talent/job-descriptions/infrastructure.md

```diff
@@ -23,7 +23,7 @@
 
 - At source{d}, we really care about open source,which is why we contribute to projects like Terraform & CoreOS, as well as creating our own projects across different teams at source{d}.
 
-- Good knowledge of containers, kubernetes and CoreOS is important. You will also be expected to have strong knowledge of Networking, Security, Linux (Kernel, Namespaces, cgroups). Finally, backend skills in at least one language and caring about best development practices, are required.
+- We are looking for someone with strong background in Linux and knowledge of Networking to some extend. Finally, knowledge of at least one backend/scripting language and caring about best development practices are required. 
```

Comment:
I've rephrased this adding a couple of things that are interesting. Let me know what you think.

We are looking for someone with a strong Linux background, good Networking knowledge, a passion for automation and experience working at scale. Finally, knowledge of at least one backend/scripting language and caring about best development practices are required